src/code/engine_library/plotLoopPopulationVPCforPhysiology.m

```diff
@@ -61,7 +61,7 @@
 for iSet = 1:nOuterLoop
         
     %initialize new sheet
-    header = sprintf('Physiology of %s',populationReportName{iSet});
+    header = sprintf('Demography of %s vs. %s',populationReportName{iSet},populationReportName{iSet});
```

Comment:
wrong. Must be:
`sprintf('Demography of %s vs. simulated %s',PopRunSet(iSet).reportName,populationReportName{iSet})`